server/src/main/java/io/spine/server/entity/RecordBasedRepository.java

```diff
@@ -269,37 +259,28 @@ protected E findOrCreate(I id) {
         return result;
     }
 
-    /**
-     * Loads all the entities in this repository.
-     *
-     * <p>Note: The storage must be assigned before calling this method.
-     *
-     * @return all the entities in this repository
-     * @see #loadAll(Iterable)
-     */
-    public Iterator<E> loadAll() {
-        Iterator<EntityRecord> records = loadAllRecords();
-        Function<EntityRecord, E> toEntity = storageConverter().reverse();
-        Iterator<E> result = transform(records, toEntity::apply);
-        return result;
-    }
-
     /**
      * Obtains iterator over all present {@linkplain EntityRecord entity records}.
      *
+     * <p>The resulting entity states have only the specified in the {@code mask} fields. However,
+     * if the {@code mask} is empty, all the fields are retrieved.
+     *
      * @return an iterator over all records
      */
     @Internal
-    public Iterator<EntityRecord> loadAllRecords() {
+    public Iterator<EntityRecord> loadAllRecords(ResponseFormat format) {
+        checkNotNull(format);
         RecordStorage<I> storage = recordStorage();
-        Iterator<EntityRecord> records = storage.readAll();
+        Iterator<EntityRecord> records = storage.readAll(format);
         return records;
     }
 
     /**
      * Finds the entities passing the given filters and applies the given {@link FieldMask}
-     * to the results. A number of elements to retrieve can be limited by {@link Pagination}.
-     * OrderBy in which to look for and return results in is specified by the {@link OrderBy}.
+     * to the results.
+     *
+     * <p>A number of elements to retrieve can be limited to a certain number. Order in
+     * which to look for and return results in is specified by the {@link OrderBy}.
```

Comment:
Please remove `in` (it's the second one in this sentence and therefore redundant).